sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -109,10 +108,9 @@ client.create(chatThread: thread) { result, _ in
             }
         }
         guard let thread = threadId else { fatalError("ThreadId not found.") }
-        print("<- SUCCESS! in createThread. Thread id= \(thread)")
+        //TODO: Take further action
 
     case let .failure(error):
-        print("!! FAIL! in createThread.")
         fatalError(error.message)
```

Comment:
Generally in a real app, you wouldn't want to use fatalError since that will make the app crash. You might replace this with another TODO (i.e: Display error message)